gammapy/estimators/excess_map.py

```diff
@@ -82,6 +82,8 @@ class ExcessMapEstimator(Estimator):
     apply_mask_fit : Bool
         Apply a mask for the computation.
         A `~gammapy.datasets.MapDataset.mask_fit` must be present on the input dataset
+    correlate_off : Bool
+        Correlate OFF events in the case of a MapDatasetOnOff
```

Comment:
You could add a line in the docstring before the parameters explaining the role of this parameter.
Something like, "Some OFF estimation techniques, such as the ring background and adaptive ring background, will yield already correlated `counts_off` maps. They should therefore not be correlated again and the `correlated_off` parameter must be set to False.

Reply:
Ok sure I will do that.